### PR TITLE
add run examples scripts

### PR DESCRIPTION
**Pull request by joaomcteixeira**
Add script to run examples in a row. Why not have this here for now? :relaxed:

Usage (from within the `examples/` folder):

```bash
./run_examples.sh 0  # -> runs all examples regardless of errors
./run_examples.sh 1  # -> run all examples but will stop if an error happens
```

### CODE REVIEW

**Review by rvhonorato — APPROVED** [review]

**Review by amjjbonvin — APPROVED** [review]
You could take 0 as default in case of no argument given...

File: examples/run_examples.sh [hunk 1]
```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Script used to run all examples at once.
+# Does not parse examples automatically. The exact examples that are run
+# are defined here in the script. Run this script from inside the `examples`
+# folder.
+#
+# USAGE:
```
Comment (amjjbonvin) [inline examples/run_examples.sh:8]: I would implement a `-h` argument that returns the usage :-)

**Review by amjjbonvin — CHANGES_REQUESTED** [review]

**Comment by joaomcteixeira** [comment]
@amjjbonvin made it in python at the end.

`python run_examples.py -h`

you need to have the `haddock3` environment activated. But that also happened with the previous `sh` version. Any new examples must be added to the pipe manually by editing the script. I did not make an official CLI out of this because it only makes sense to run it inside `examples/` and because of the hardcoded examples (it should be this way).

**Review by amjjbonvin — APPROVED** [review]

File: examples/run_examples.sh (hunk 1 above)
Comment (joaomcteixeira) [reply examples/run_examples.sh]: :dark_sunglasses:

**Comment by rvhonorato** [comment]
Python 1 x Bash 0! 😉

**Comment by amjjbonvin** [comment]
What about the carbon footprint of the two? Which version uses more characters / lines to do the same?

**Comment by joaomcteixeira** [comment]
:rofl: i needed to add more chars because of the `-h` option :stuck_out_tongue_closed_eyes: definitively I wrote the python version better.

**Pull request merged**